timon/tests/test_db.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+"""
+#############################################################################
+Copyright : (C) 2023 by Teledomic.eu All rights reserved
+
+Name:         timon.test.test_db
+
+Description:  some unit tests for checking parts of an http probe
+
+#############################################################################
+"""
+from datetime import datetime
+from pathlib import Path
+import time
+from unittest import TestCase
+
+from timon.db.store import get_store
+
+
+class ConfigTestCase(TestCase):
+    def setUp(self):
+        self.dbstore = None
+        return super().setUp()
+
+    def tearDown(self):
+        if self.dbstore:
+            self.dbstore.stop()
+        return super().tearDown()
+
+    def test_peeweesqlite_dbstore(self):
+        """
+        check that the dbstore can be initialized and rslts are correctly
+        written in db
+        """
+        sqlite_db_fname = "timon_tstsqlite.db"
+        sqlite_db_fpath = Path(sqlite_db_fname)
+        if sqlite_db_fpath.exists():
+            sqlite_db_fpath.unlink()
+        prbname = "PRBNAME0"
+        self.dbstore = dbstore = get_store(db_fpath=sqlite_db_fname)
+        dbstore.start(probenames=[prbname])
```

Comment:
add status to db store, that says it's fully initialize or ready or whatever

then

```
cnt = 0
while not status == "READY" and cnt < 100:
   time.sleep(0.05)
   cnt += 1
 ```